Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx

```diff
@@ -187,6 +190,11 @@ export default class EquipmentContainer extends React.Component<IProps, IState>
                     this._checkAttributeFilters(x, this.state.attributeFilters)
                 );
             }
+            if (this.state.equipmentTypeFilters.length > 0) {
+                filteredEquipment = filteredEquipment.filter(x => 
+                    this._checkEquipmentTypeFilters(x, this.state.equipmentTypeFilters)
```

Comment:
something to think about -- you don't need to pass `this.state.equipmentTypeFilters`, since that method is part of this class and already has access to `this.state`.  If you did, this line could become `filteredEquipment.filter(this._checkEquipmentTypeFilters)`.